assets/helpers/formValidation.js

```diff
@@ -5,15 +5,50 @@
 // src/main/scala/play/api/data/validation/Validation.scala#L81
 // but with minor modification (last * becomes +) to enforce at least one dot in domain.  This is
 // for compatibility with Stripe
+import { config } from 'helpers/contributions';
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import type { ContributionType, OtherAmounts, SelectedAmounts } from 'helpers/contributions';
+
 export const emailRegexPattern = '^[a-zA-Z0-9\\.!#$%&\'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)+$';
 
-export const isNotEmpty: string => boolean = input => !!input && input.trim() !== '';
-export const isValidEmail: string => boolean = input => new RegExp(emailRegexPattern).test(input);
+export const isNotEmpty: (string | null) => boolean = input => !!input && input.trim() !== '';
+export const isValidEmail: (string | null) => boolean = input => !!input && new RegExp(emailRegexPattern).test(input);
 export const isLargerOrEqual: (number, string) => boolean = (min, input) => min <= parseFloat(input);
 export const isSmallerOrEqual: (number, string) => boolean = (max, input) => parseFloat(input) <= max;
 export const maxTwoDecimals: string => boolean = input => new RegExp('^\\d+\\.?\\d{0,2}$').test(input);
 
-export const checkFirstName: string => boolean = isNotEmpty;
-export const checkLastName: string => boolean = isNotEmpty;
+export const checkFirstName: (string | null) => boolean = isNotEmpty;
```

Comment:
this is in anticipation of removing them from one-off?

Reply:
No, it's because the type of firstName is `string | null`, so it seemed clearer to be explicit about what types it could take (it didn't actually throw an error previously, but I prefer this)